FHIR-Converter/client/src/core/common/enum/template-type.ts

```diff
@@ -5,5 +5,5 @@
 
 export enum TemplateType { 
 	ccda = 'C-CDA', 
-	hl7v2 = 'HL7v2' 
+	hl7v2 = 'HL7v2'
```

Comment:
What's the difference between MetadataType and TemplateType?

Reply:
MetadataType is the words used in the engine, which is in lowercase. Templatetype is used to display to users in VS Code extension, which is uppercase, more formal

Reply:
Shown in the discussion below, since the sample templates are treated as "default" templates for JSON, we need to add JSON for TemplateType as well.

Reply:
> Shown in the discussion below, since the sample templates are treated as "default" templates for JSON, we need to add JSON for TemplateType as well.

By offline discussion, we are not going to support "default" templates for JSON for now.